streaming/src/main/scala/io/gearpump/streaming/task/Task.scala

```diff
@@ -183,4 +186,12 @@ abstract class Task(taskContext : TaskContext, userConf : UserConfig) extends Ta
     case msg =>
       LOG.error("Failed! Received unknown message " + "taskId: " + taskId + ", " + msg.toString)
   }
+}
+
+class TypedTask[T](val typedTask: T, taskContext : TaskContext, userConf : UserConfig) extends Task(taskContext, userConf) {
```

Comment:
@manuzhang 
See [PR](https://github.com/gearpump/gearpump-examples/pull/16) in gearpump-examples